app.py

```diff
@@ -1998,6 +1998,82 @@ def unlink_globus_identities():
 
     return render_template('instructions/unlink_globus_identities.html', data = context)
 
+def get_connection_keys():
+    return ['other_component', 'other_organization', 'other_role', 'access_requests',
+            'globus_identity', 'google_email', 'github_username', 'slack_username', 'protocols_io_email',
+            'website', 'orcid', 'pm', 'pm_name', 'pm_email']
+
+# Get a list of all approved members (not including admins)
+def get_all_members_with_all_info():
+    members = list()
+    # Order members by ID DESC
+    wp_users = WPUser.query.order_by(WPUser.id.desc()).all()
+    for user in wp_users:
+        # Check if this target user is a member (capabilities will be empty dict if not member role)
+        capabilities = next((meta for meta in user.metas if (meta.meta_key == wp_db_table_prefix + 'capabilities') and ('member' in meta.meta_value)), {})
```

Comment:
I think we should also include the `administrator` to the download, not only `member` role. 

Let's also remove the "(not including the Administrators)" wording. And you'll also need to add the `administrator` role to https://github.com/sennetconsortium/member-ui/blob/libpitt/member-info-4/app.py#L1224 for consistency.

![image](https://user-images.githubusercontent.com/195873/200910599-386e80bf-7a64-4967-8426-6a93a3836857.png)